Packs/Zoom/Integrations/Zoom/Zoom.yml

```diff
@@ -533,6 +533,739 @@ script:
+      type: string
+    - name: include_deleted_and_edited_message
+      description: Include deleted and edited messages. Default false.
+      required: false
+      defaultValue: false
+      type: boolean
+    - name: search_type
+      description: >
+        Allowed: message ┃ file
+        The type of search:
+        message — Search messages.
+        file — Search files.
+      required: false
+      type: string
+      auto: PREDEFINED
+      predefined:
+      - message
+      - file
+    - name: search_key
+      description: Key for search in messages.
+      required: false
+      type: string
+    - name: exclude_child_message
+      description: >
+        Default: false
+        This parameter is used to exclude returning all child messages in a chat, leaving only the parent messages.
+      required: false
+      type: boolean
+      defaultValue: false
+    - name: next_page_token
+      description: >
+        The next page token paginates through a large set of results. A next page token is returned whenever the set of available results exceeds the current page size.
+        This token's expiration period is 15 minutes.
+      required: false
+      type: string
+    - name: page_size
+      description: Number of records returned from a single API call. Default 50.
+      required: false
+      type: number
+    - name: page_number
+      description: Which page of results to return. Default 1.
+      required: false
+      type: number
+    - name: limit
+      description: Total amount of results to show.
+      required: false
+      type: number
+    outputs:
+    - contextPath: Zoom.ChatMessage.date
+      type: date
+      description: The queried date value.
+    - contextPath: Zoom.ChatMessage.id
+      description: the channel's unique identifier.
+      type: string
+    - contextPath: Zoom.Channel.jid
+      description: The channel JID.
+      type: string
+    - contextPath: Zoom.Channel.name
+      description: The channel name.
+      type: string
+    - contextPath: Zoom.Channel.type
+      type: string
+      description: >
+        The type of the channel. The value can be one of the following:
+        1: Private channel. In this type of channel, members must be invited to join a channel.
+        2: Private channel with members that belong to one Zoom account. Members in this channel should be invited and the members should be from the same organization.
+        3: Public channel. Anyone can search for this channel and join the channel.
+        4: Group chat. This is an instant channel which can be created by adding members to a new chat.
+        Allowed: 1┃2┃3┃4
+    - contextPath: Zoom.ChatMessage.date
+      type: date
+      description: The queried date value.
+    - contextPath: Zoom.ChatMessage.from
+      type: date-time
+      description: The queried from value. (Returned only if the from query parameter is used)
+    - contextPath: Zoom.ChatMessage.messages
+      type: array of object
```

Comment:
Is this type correct?  (array of object)
I've never seen that as a type before.